docs/sources/panels/visualizations/node-graph.md

```diff
@@ -8,13 +8,13 @@ weight = 850
 
 > **Note:** This panel is currently in beta. Expect changes in future releases.
 
-The _Node graph_ can visualize directed graphs or networks. It uses directed force layout to effectively position the nodes so it can help with displaying complex infrastructure maps, hierarchies or execution diagrams.
+The _Node graph_ can visualize directed graphs or networks. It uses directed force layout to effectively position the nodes, so it can help with displaying complex infrastructure maps, hierarchies or execution diagrams.
```

Comment:
```suggestion
The _Node graph_ can visualize directed graphs or networks. It uses a directed force layout to effectively position the nodes, so it can display complex infrastructure maps, hierarchies, or execution diagrams.
```